### PR TITLE
Show display names for reposts

### PR DESCRIPTION
**Pull request by garritfra**
On the timeline, the app currently displays the username (E.g. `foo@mastodon.social`) of someone who reposted a toot.

This PR aims to show the display name (E.g. `Foo Bar 🕹`) of that user.

[This matrix conversation](https://matrix.to/#/!PWkUGGpeMZWDPmdfRE:matrix.org/$16103634431sblev:matrix.slashdev.space?via=matrix.org&via=kde.org&via=converser.eu) gives some more context.

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java [hunk 1]
```diff
@@ -73,10 +75,11 @@ protected void setupWithStatus(StatusViewData.Concrete status,
 
     }
 
-    private void setRebloggedByDisplayName(final String name) {
+    private void setRebloggedByDisplayName(final CharSequence name, final StatusViewData.Concrete status) {
         Context context = statusInfo.getContext();
-        String boostedText = context.getString(R.string.status_boosted_format, name);
-        statusInfo.setText(boostedText);
+        CharSequence boostedText = context.getString(R.string.status_boosted_format, name);
```
Comment (Tak) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java:80]: minor: I would maybe push the unicode wrap down to here

**Review by Tak — COMMENTED** [review]
`StatusBaseViewHolder` also needs to be updated, doesn't it?

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java (hunk 1 above)
Comment (garritfra) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusViewHolder.java]: Good point. Fixed in 221537b.

**Comment by garritfra** [comment]
> StatusBaseViewHolder also needs to be updated, doesn't it?

@Tak Do you mean here?  https://github.com/tuskyapp/Tusky/blob/9cdf486ceb36852e0c07af4678eeab934fc39a98/app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java#L804

As far as I can tell, the only class needing this logic is the `StatusViewHolder`. All other subclasses don't rely on this information.

image

**Comment by connyduck** [comment]
We only show the reblogging information in the timeline, so this is fine. Showing it on the details screen as well is something for a future improvement.
thx @garritfra

**Pull request merged**